### DISCUSSION TITLE
RD on windows: .\\pipe\\containerd-containerd: The system cannot find the file specified

### DISCUSSION THREAD

**Discussion by mirzaprangon** [rancher-sandbox/rancher-desktop#2870] [Q&A] [answered]
Hello
I have already posted on slack about a week ago, but as there was no solution. So I am asking for help here.

I am migrating from docker. Installed RD 1.5.1 on windows. But running command like `nerdctl run -it --rm alpine` fails with the following error
```
time="2022-09-06T09:51:47+09:00" level=fatal msg="cannot access containerd socket \"\\\\\\\\.\\\\pipe\\\\containerd-containerd\": open \\\\.\\pipe\\containerd-containerd: The system cannot find the file specified."
```
 Output of `rdctl list-settings`
```
{
  "version": 4,
  "kubernetes": {
    "version": "1.24.4",
    "memoryInGB": 2,
    "numberCPUs": 2,
    "port": 6443,
    "containerEngine": "containerd",
    "checkForExistingKimBuilder": false,
    "enabled": false,
    "WSLIntegrations": {
      "Ubuntu-20.04": false
    },
    "options": {
      "traefik": true,
      "flannel": true
    },
    "suppressSudo": false,
    "hostResolver": true
  },
  "portForwarding": {
    "includeKubernetesServices": false
  },
  "images": {
    "showAll": true,
    "namespace": "buildkit"
  },
  "telemetry": true,
  "updater": true,
  "debug": true,
  "pathManagementStrategy": "notset"
}

```
I will also attach the logs I got after enabling debug mode. (I stopped RD before archiving the logs)
Can anyone help to resolve the situation?
[logs.zip](https://github.com/rancher-sandbox/rancher-desktop/files/9502114/logs.zip)

**Comment by mook-as** [comment, marked as answer]
Sorry we missed your message on Slack!

Are you using a Windows-native `nerdctl`, instead of the stub we ship (that is designed to talk to the containerd in our WSL VM)? Nothing we do should touch `\\.\pipe\containerd-containerd`.

Please check (if in PowerShell, I believe it's `Get-Command nerdctl`).

**Reply by mirzaprangon** [reply]
You are correct! I added the windows native nerdctl a long time ago, and totally forgot about it. Running the `Get-Command nerdctl` showed the path to the native nerdctl path.

Removed the native nerdctl path from the environment variable. Now running `Get-Command nerdctl` shows the path for the executable that ships with Rancher Desktop.

Tried `nerdctl run...` and `nerdctl compose up` with my existing compose files, and it is working!

Thanks!
